elasticsearch/sgconfig/roles.yml

```diff
@@ -135,11 +136,13 @@ project_user:
   cluster:
     - CLUSTER_COMPOSITE_OPS_RO
     - MONITOR
+    - SEARCH #required for index level metrics
```

Comment:
Maybe search should be added to another group?  https://github.com/openshift/origin-aggregated-logging/blob/ea15554f30d5fcff6806d33af222e95aea9c85d4/elasticsearch/sgconfig/action_groups.yml#L127

Reply:
I will add `indices:/data/read/search` to `CLUSTER_COMPOSITE_OPS_RO`